### PR TITLE
CT5UP-468-dev-server-polls-ip 

### PR DESCRIPTION
**Pull request by dylanhamersztein**
dev server now checks if the computer's IP has changed, re-compiles the app, and prints the new address.

### CODE REVIEW

File: express-dev-server/server.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+const Timers = require("timers");
```
Comment (briandipalma) [inline express-dev-server/server.js]: No need I think the `timers` functions are global.

File: express-dev-server/server.js [hunk 2]
```diff
@@ -11,19 +12,38 @@ const getPort = require("./get-port");
 
 const { hot } = parseArgs(process.argv.slice(2));
 
+const ipCheckInterval = 300000;
+var checkIP = true;
```
Comment (briandipalma) [inline express-dev-server/server.js]: `let`

File: express-dev-server/server.js [hunk 3]
```diff
@@ -11,19 +12,38 @@ const getPort = require("./get-port");
...
+var currentIP = address.ip();
```
Comment (briandipalma) [inline express-dev-server/server.js]: `let`

**Comment by briandipalma** [comment]
I don't understand this PR, what is it for?

**Comment by dylanhamersztein** [comment]
@briandipalma Daniel mentioned that Steve's IP changed between the time he sent Dan a link to a dev server and the time Daniel opened it, so I guess this is a quality of life thing!

**Comment by briandipalma** [comment]
I'm not a fan of this tbh, it seems like a minor benefit for a rare edge case. @thecapdan @stevesouth do you really want this?

**Comment by thecapdan** [comment]
it is a bit of an edge case. @dylanhamersztein is working on an alternative method without polling, he's in  a strict timebox;)

**Comment by thecapdan** [comment]
this is looking like more effort than its worth. will decline for now and we can revisit if needed

**Pull request closed without merging**